Backpack/Font/Classes/Generated/BPKFont.m

```diff
@@ -228,8 +230,40 @@ + (UIFont *)fontForStyle:(BPKFontStyle)style fontManager:(BPKFontManager *)fontM
     }
 }
 
-+ (NSNumber *)letterSpacingForStyle:(BPKFontStyle)style {
++ (BPKFontStyle)newStyleMappingForStyle:(BPKFontStyle)style {
     switch (style) {
+        case BPKFontStyleTextBase: 
```

Comment:
should we also map the actually deprecated styles to their replacement (e.g. the heavy styles)?